src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/ActiveDirectorySchemaClassCollection.cs

```diff
@@ -286,7 +286,7 @@ protected override void OnClearComplete()
             }
         }
 
-        protected override void OnInsertComplete(int index, object value)
+        protected override void OnInsertComplete(int index, object? value)
```

Comment:
technically they shouldn't be nullable, it's enforced through base class so that's why they're marked this way. Since this is protected I doubt this will matter anyway for any public APIs.

Reply:
>  it's enforced through base class so that's why they're marked this way

Yes, that is why i said to suppress it, for me ti should be suppressed, but i will let you and @stephentoub decide

> Since this is protected I doubt this will matter anyway for any public APIs.

Even it is protected as the type is public it is still exposed at some level, I see them in ref too. https://github.com/dotnet/runtime/blob/4ff89aa70b102980034030aa32511d42c2d13c59/src/libraries/System.DirectoryServices/ref/System.DirectoryServices.cs#L710-L712